src/plugins/DistanceMeasurementsPlugin/DistanceMeasurement.js

```diff
@@ -784,6 +785,25 @@ class DistanceMeasurement extends Component {
         return this._targetVisible;
     }
 
+    /**
+     * Sets if the measurement is adjusted based on rotation
+     *
+     * @type {Boolean}
+     */
+    set useRotationAdjustment(value) {
+        value = value !== undefined ? Boolean(value) : this.plugin.useRotationAdjustment;
```

Comment:
Using `this.plugin.useRotationAdjustment` as a fallback value of `_useRotationAdjustment` is a pretty significant semantic inconsistency related to other setters - what is the reason to do that?
This setter is a part of an external API, we should be cautious about introducing such unexpected behaviors exposed to the library users.

Reply:
Ok I see other setters in this particular class are implemented in a similarly peculiar way.